include/orc/orc.hpp

```diff
@@ -18,8 +18,7 @@
 
 struct odrv_report {
     std::string_view _symbol;
-    die _a;
-    die _b;
+    const die* _die{nullptr};
```

Comment:
Cool. Maybe should be `_die_root`? When I first read it I was looking for the 2nd one.